grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -681,7 +681,6 @@ class ReportService implements ApplicationContextAware {
             [
                     productCode          : it.productCode,
                     productName          : product?.displayNameWithLocaleCode,
-                    translatedProductName : product?.translatedName,
```

Comment:
is there a reason it has been removed, not replaced by the `displayName`?

Reply:
Yes, it should be replaced, thanks for spotting that.
Not sure how I missed that :confused: